### PR TITLE
[DB-6452]Export data status command for exporting data using Debezium

### PR DESCRIPTION
**Pull request by priyanshi-yb**
Fixes - [DB-6452](https://yugabyte.atlassian.net/browse/DB-6452)

### CODE REVIEW

File: yb-voyager/cmd/exportDataStatusCommand.go
Comment (sanyamsinghal) [inline yb-voyager/cmd/exportDataStatusCommand.go:28]: nit: rename `setUseDebeziumFlag()` to something like `checkDebezium()` and directly use that at `line number: 28`
That function itself can exit in case of a error

File: yb-voyager/cmd/exportDataStatusCommand.go
Comment (sanyamsinghal) [inline yb-voyager/cmd/exportDataStatusCommand.go:67]: Try reusing this function: `status.InProgressTableSno()`

File: yb-voyager/cmd/exportDataStatusCommand.go
Comment (sanyamsinghal) [inline yb-voyager/cmd/exportDataStatusCommand.go:66]: nit: create var of type exportTableMigStatusOutputRow and use that only instead of extra var

**Review by sanyamsinghal — APPROVED** [review]
LGTM provided some minor review comments. 
Respond to them before merging the PR

File: yb-voyager/cmd/exportDataStatusCommand.go
Comment (sanyamsinghal) [inline yb-voyager/cmd/exportDataStatusCommand.go:65]: you can remove this comment now. Line 64 is self explanatory

File: yb-voyager/cmd/exportDataStatusCommand.go
Comment (amit-yb) [inline yb-voyager/cmd/exportDataStatusCommand.go:57]: No need of this variable. Directly use the `status.Tables` in line 59.

File: yb-voyager/cmd/exportDataStatusCommand.go
Comment (amit-yb) [inline yb-voyager/cmd/exportDataStatusCommand.go:58]: How about naming it as `rows`? https://research.swtch.com/names

File: yb-voyager/cmd/exportDataStatusCommand.go
Comment (amit-yb) [inline yb-voyager/cmd/exportDataStatusCommand.go:60]: Can we include exported row count in the output (if we're using debezium)?

File: yb-voyager/cmd/exportDataStatusCommand.go
Comment (amit-yb) [inline yb-voyager/cmd/exportDataStatusCommand.go:64]: Call `status.InProgressTableSno()` only once--before the loop.

**Review by amit-yb — COMMENTED** [review]
A few minor comments.

File: yb-voyager/cmd/exportDataStatusCommand.go
Comment (priyanshi-yb) [reply yb-voyager/cmd/exportDataStatusCommand.go:60]: Yes that can be done, but like with ora2pg we don't output that so uniformity won't be there. If that is not a problem, I can add it?

File: yb-voyager/cmd/exportDataStatusCommand.go
Comment (priyanshi-yb) [reply yb-voyager/cmd/exportDataStatusCommand.go:183]: Will remove this

File: yb-voyager/cmd/exportDataStatusCommand.go
Comment (priyanshi-yb) [reply yb-voyager/cmd/exportDataStatusCommand.go:26]: need to use flag itself now, as it requires in display function as well as we are now displaying export rows in case of dbzm as well

**Review by amit-yb — APPROVED** [review]
LGTM.

**Pull request merged**
